modules/appeals_api/app/views/appeals_api/decision_review_mailer/_evidence_table.html.erb

```diff
@@ -0,0 +1,64 @@
+<h2><span class="title"><%= title %> submissions by status and count</span></h2>
+<table>
+  <thead>
+  <tr>
+    <th>Status</th>
+    <th>Count</th>
+  </tr>
+  </thead>
+  <tbody>
+  <% status_and_count.each do |status, count| %>
+    <tr>
+      <td class="right-align"><%= status %></td>
+      <td class="right-align"><%= count %></td>
+    </tr>
+  <% end %>
+  </tbody>
+</table>
+
+<h2><span class="title">Faulty <%= title %></span></h2>
```

Comment:
is the thought here that other data sources with different titles would get used for this partial, as well?

Reply:
Yes it was. I meant to update both our NOD Evidence and SC Evidence tables to use this new evidence table, but looks like I only did it for NOD Evidence. Updating!